src/tunnel.c

```diff
@@ -46,7 +46,12 @@
 #elif HAVE_UTIL_H
 #include <util.h>
 #endif
+#if HAVE_LIBUTIL
+#include <libutil.h>
+#endif
+#if HAVE_TERMIOS_H
```

Comment:
<termios.h> is POSIX and should be available on all supported platforms. While the #ifdef does't really hurt, perhaps we should remove it for simplicity.

Reply:
Out of curiosity, what is <libutil.h> for?

Reply:
since we didn't have to include termios.h explicitly on other platforms than FreeBSD so far, I have included it in the `#ifdef` section, honestly without further investigations if this header should always be there.

The reason for both headers is the [synopsis of forkpty](https://www.freebsd.org/cgi/man.cgi?query=forkpty)

Reply:
Header <termios.h> is included implicitly on Linux, it just needs to be included explicitly on all platforms. Build should fail without <termios.h>. I suggest removing the `#ifdef` from the source file and have _autoconf_ fail if <termios.h> is missing.

Reply:
Indeed it looks like FreeBSD is not compliant with recent versions of POSIX which specify _fokrpty_() is defined in `<termios.h>`.

I checked FreeBSD's [`<termios.h>`](https://github.com/freebsd/freebsd/blob/master/include/termios.h) does not define _forkpty_(), [`<libutil.h>`](https://github.com/freebsd/freebsd/blob/master/lib/libutil/libutil.h) does.